src/Constants/index.js

```diff
@@ -0,0 +1,83 @@
+const CONSTANTS = {
+  LOTTO: {
+    INCLUSIVE_LOWER_NUMBER: 1,
+    INCLUSIVE_UPPER_NUMBER: 45,
+    NUMBER_COUNT: 6,
+
+    CAHCE: {
+      MINIMUM: 1_000,
+      BASE_UNIT: 1_000,
+    },
+
+    WINNING_PRICE: {
+      FIFTH: 5_000,
+      FOURTH: 50_000,
+      THIRD: 1_500_000,
+      SECOND: 30_000_000,
+      FIRST: 2_000_000_000,
+    },
+
+    FORMAT: {
+      SQUARE_BRACKET_OPEN: '[',
+      DELIMITER: ', ',
+      SQUARE_BRACKET_CLOSE: ']',
+    },
+  },
+
+  VALIDATION: {
+    MESSAGE: {
+      NULL: '빈 값이 아닌 다른 값을 입력해주세요.',
+      NOT_NUMBERIC: '숫자를 입력해주세요.',
+      UNDER_THOUSAND: '1,000원 이상 입력해주세요.',
+      NOT_DIVIDED_THOUSAND: '1,000원 단위로만 입력해주세요.',
+      NOT_FORMATTED: '숫자와 쉼표(,)로만 입력해주세요.',
+      NOT_SIX_LENGTH: '6개의 숫자를 입력해주세요.',
```

Comment:
저도 코드리뷰하면서 알게된 점인데 상수 안에 상수화도 고려해 보시면 좋을 것 같습니다. 
```suggestion
      NOT_SIX_LENGTH: `${LOTTO.NUMBER_COUNT}개의 숫자를 입력해주세요.`,
```

이렇게요.

Reply:
앞으론 끌어다쓸 상수가 있다면 그렇게 해봐야겠어요! 감사합니다 :)